plugins/GSdx/Renderers/DX11/GSRendererDX11.cpp

```diff
@@ -501,6 +501,114 @@ void GSRendererDX11::EmulateChannelShuffle(GSTexture** rt, const GSTextureCache:
 	}
 }
 
+void GSRendererDX11::EmulateBlending()
+{
+	// Partial port of OGL SW blending. Currently only works for accumulation blend or when
+	// m_sw_blending >= ACC_BLEND_HIGH_D3D11 since it has not been fully tested. There is no check
+	// currently for primitive overlap like in OGL but it should give better results for post-processing effects
+	// that are done by non-overlapping sprites.
+	const GIFRegALPHA& ALPHA = m_context->ALPHA;
+	bool sw_blending         = false;
+
+	m_om_bsel.abe = PRIM->ABE || (PRIM->AA1 && m_vt.m_primclass == GS_LINE_CLASS);
```

Comment:
I think we can do a return like on gl here for an optimization.

```
	if (PRIM->ABE || (PRIM->AA1 && m_vt.m_primclass == GS_LINE_CLASS))
		m_om_bsel.abe = 1;
	else
		return;
```

Reply:
Actually, I think this might be a good idea to do now. Aside from the optimization, if blending is not even enabled then we shouldn't be making changes to the pixel shader settings. It might even give wrong results: if ABE = false but A, B, C, D are non zero we might end up doing SW blending in the shader when we shouldn't (IDK for sure because I have not tested and seems as if some games keep ABE = true by default).